client/src/components/page/StatsPage.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import { STATS } from 'constants/links.js';
+
+/** Renders the /example-page page. */
+class StatsPage extends Component {
+  state = {
+    content: null
+  };
+
+  componentDidMount() {
+    this.fetchStatsPage();
+  }
+
+  /** Fetches the content for the example page. */
+  fetchStatsPage() {
+    fetch(STATS)
```

Comment:
I would make this a different link than STATS.

You should make two links.
STATS = "/stats"
STATS_SERVLET = "/api/stats"

Here you should use STATS_SERVLET because this fetch is communicating with the servlet.